src/Sulu/Bundle/MediaBundle/Tests/app/config/config.yml

```diff
@@ -8,6 +8,9 @@ sulu_media:
         response_headers:
             Expires: "+1 month"
 
+framework:
+    assets: ~
```

Comment:
Else the 404 exception_full.html.twig from twig-bundle does not work on version 3.0.0 in the tests.